web3.js/module.d.ts

```diff
@@ -966,6 +966,54 @@ declare module '@solana/web3.js' {
     ): AuthorizeNonceParams;
```

Comment:
Please export `SYSVAR_INSTRUCTIONS_PUBKEY` in this file and in the flow file

Reply:
Thanks! Late addition